opcodes/riscv-opc.c

```diff
@@ -2034,14 +2034,14 @@ const struct riscv_opcode riscv_opcodes[] =
 {"cv.sdotup.b",           0, INSN_CLASS_COREV_SIMD, "d,s,t", MATCH_CV_SDOTUP_B, MASK_CV_SDOTUP_B, match_opcode, 0},
 {"cv.sdotup.sc.h",        0, INSN_CLASS_COREV_SIMD, "d,s,t", MATCH_CV_SDOTUP_SC_H, MASK_CV_SDOTUP_SC_H, match_opcode, 0},
 {"cv.sdotup.sc.b",        0, INSN_CLASS_COREV_SIMD, "d,s,t", MATCH_CV_SDOTUP_SC_B, MASK_CV_SDOTUP_SC_B, match_opcode, 0},
-{"cv.sdotup.sci.h",       0, INSN_CLASS_COREV_SIMD, "d,s,b5", MATCH_CV_SDOTUP_SCI_H, MASK_CV_SDOTUP_SCI_H, match_opcode, 0},
-{"cv.sdotup.sci.b",       0, INSN_CLASS_COREV_SIMD, "d,s,b5", MATCH_CV_SDOTUP_SCI_B, MASK_CV_SDOTUP_SCI_B, match_opcode, 0},
+{"cv.sdotup.sci.h",       0, INSN_CLASS_COREV_SIMD, "d,s,b8", MATCH_CV_SDOTUP_SCI_H, MASK_CV_SDOTUP_SCI_H, match_opcode, 0},
+{"cv.sdotup.sci.b",       0, INSN_CLASS_COREV_SIMD, "d,s,b8", MATCH_CV_SDOTUP_SCI_B, MASK_CV_SDOTUP_SCI_B, match_opcode, 0},
 {"cv.sdotusp.h",          0, INSN_CLASS_COREV_SIMD, "d,s,t", MATCH_CV_SDOTUSP_H, MASK_CV_SDOTUSP_H, match_opcode, 0},
 {"cv.sdotusp.b",          0, INSN_CLASS_COREV_SIMD, "d,s,t", MATCH_CV_SDOTUSP_B, MASK_CV_SDOTUSP_B, match_opcode, 0},
 {"cv.sdotusp.sc.h",       0, INSN_CLASS_COREV_SIMD, "d,s,t", MATCH_CV_SDOTUSP_SC_H, MASK_CV_SDOTUSP_SC_H, match_opcode, 0},
 {"cv.sdotusp.sc.b",       0, INSN_CLASS_COREV_SIMD, "d,s,t", MATCH_CV_SDOTUSP_SC_B, MASK_CV_SDOTUSP_SC_B, match_opcode, 0},
-{"cv.sdotusp.sci.h",      0, INSN_CLASS_COREV_SIMD, "d,s,b5", MATCH_CV_SDOTUSP_SCI_H, MASK_CV_SDOTUSP_SCI_H, match_opcode, 0},
-{"cv.sdotusp.sci.b",      0, INSN_CLASS_COREV_SIMD, "d,s,b5", MATCH_CV_SDOTUSP_SCI_B, MASK_CV_SDOTUSP_SCI_B, match_opcode, 0},
+{"cv.sdotusp.sci.h",      0, INSN_CLASS_COREV_SIMD, "d,s,b8", MATCH_CV_SDOTUSP_SCI_H, MASK_CV_SDOTUSP_SCI_H, match_opcode, 0},
+{"cv.sdotusp.sci.b",      0, INSN_CLASS_COREV_SIMD, "d,s,b8", MATCH_CV_SDOTUSP_SCI_B, MASK_CV_SDOTUSP_SCI_B, match_opcode, 0},
```

Comment:
Is this correct? The spec for `cv.sdotusp[.sc,.sci].h` and `cv.sdotusp[.sc,.sci].b` says the second source operand is signed.  It is the first source operation, which is signed, but this is always a 32-bit register anyway.

Reply:
> Is this correct? The spec for `cv.sdotusp[.sc,.sci].h` and `cv.sdotusp[.sc,.sci].b` says the second source operand is signed. It is the first source operation, which is signed, but this is always a 32-bit register anyway.

I have reverted the commit so that cv.sdotusp[.sc,.sci][.h,.b] is using a signed immediate.

Reply:
Fix looks good. Thanks.